Uni_life_tracker/app/src/main/java/com/generic/ult/SignUpActivity.java

```diff
@@ -1,10 +1,13 @@
 package com.generic.ult;
 
+import android.annotation.SuppressLint;
 import android.content.Intent;
+import android.content.SharedPreferences;
+import androidx.preference.PreferenceManager;
```

Comment:
It shows me that there is an error with this import statement so everything related to Preference Manager has an erorr

Reply:
Weird. Maybe the dependencies were not included into the build.gradle file?